### PR TITLE
Add recipe for python package "holidays"

### PR DESCRIPTION
**Pull request by bletham**

### CODE REVIEW

File: recipes/holidays/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - python-dateutil
+    - six
```
Comment (beckermr) [inline recipes/holidays/meta.yaml]: Only dependencies need to install the package should appear here. This is anything needed by the setup.py.

**Comment by bletham** [comment]
@beckermr thanks, that should be fixed now.

**Comment by rpanai** [comment]
Is it possible to automatically update this recipe? [holidays](https://github.com/dr-prodigy/python-holidays) was recently updated with more countries.

**Comment by bletham** [comment]
@rpanai this has to be done on https://github.com/conda-forge/holidays-feedstock/ . It looks like the automatic upgrade isn't working (https://github.com/conda-forge/holidays-feedstock/pull/5), so someone is going to have to update the recipe (it looks like the dependencies may have changed with 0.10.1).

**Comment by rpanai** [comment]
@bletham could you lead me to how to do it? I never work with conda-forge before.

**Comment by bletham** [comment]
Generally the instructions are here: https://conda-forge.org/#update_recipe , and the recipe to be updated is here: https://github.com/conda-forge/holidays-feedstock/blob/master/recipe/meta.yaml . There are some really thorough steps here: https://pysteps.readthedocs.io/en/latest/developer_guide/update_conda_forge.html . If you are able to update this that'd be fantastic and I can definitely review the PR!

**Comment by rpanai** [comment]
@bletham I just created a PR. We just needed to add `convertdate`.

**Pull request merged**